SPECS/elfutils/elfutils.spec

```diff
@@ -2,7 +2,7 @@
 Summary:	A collection of utilities and DSOs to handle compiled objects
 Name:		elfutils
 Version:	0.176
-Release:	3%{?dist}
+Release:	4%{?dist}
 License:	GPLv3+ and (GPLv2+ or LGPLv3+)
```

Comment:
Nit: while you're here, could you also check the license and add `License verified` to the changelog?

Reply:
Sure. Updated.